template.yml

```diff
@@ -25,112 +25,89 @@ Parameters:
 Resources:
   InputBucket:
```

Comment:
Would be great if we could update the input bucket name in line with the other buckets. 

  InputBucket:
    Type: AWS::S3::Bucket
    Properties:
      BucketName: !Sub ${AWS::StackName}-${AWS::AccountId}-input-bucket

Reply:
It is - it's updated on Line 28.

Reply:
Sorry. I should learn to read properly.